confluent_kafka/src/Admin.c

```diff
@@ -881,7 +891,7 @@ static PyObject *Admin_alter_configs (Handle *self, PyObject *args,
          * Parse the list of ConfigResources and convert to
          * corresponding C types.
          */
-        c_objs = alloca(sizeof(*c_objs) * cnt);
+        c_objs = malloc(sizeof(*c_objs) * cnt);
```

Comment:
What if any implications are there to allocating on the heap as opposed to the stack for these Admin operations

Reply:
*aside from needing to add free